packages/it87-dkms-git/it87-dkms-git.pacscript

```diff
@@ -31,7 +33,16 @@ package() {
 }
 
 post_upgrade() {
-  update-initramfs -u -k all
+  DKMS_NAME="${gives}"
+  DKMS_VERSION="${pkgver}"
+  DKMS_OLD_VERSION="${_pkgoldver}"
+
+  dkms remove -m "${DKMS_NAME}" -v "${DKMS_OLD_VERSION}" --all || true
+  dkms add -m "${DKMS_NAME}" -v "${DKMS_VERSION}"
+  dkms build -m "${DKMS_NAME}" -v "${DKMS_VERSION}"
+  dkms install -m "${DKMS_NAME}" -v "${DKMS_VERSION}" --force || true
```

Comment:
Why can't these be combined? Why the extra variable declarations?

Reply:
It's a template.

If you increments the pkgver, because the dkms.conf is configured by changing with $pkgver, and if you install in /usr/src/module-$pkgver, you have the old plgver module in dkms tree.

In this case, pkgver doesn't change, but it can so the PKGBUILD is already correctly setup.

Reply:
Sorry, I meant, why can't it be written like:

```bash
  dkms remove -m "${gives}" -v "${_pkgoldver}" --all || true
  dkms add -m "${gives}" -v "${pkgver}"
  dkms build -m "${gives}" -v "${pkgver}"
  dkms install -m "${gives}" -v "${pkgver}" --force || true
```

Why the extra instantiated variables?

Reply:
fixed